Tests/FoundationEssentialsTests/StringTests.swift

```diff
@@ -1009,7 +1009,7 @@ final class StringTests : XCTestCase {
     func test_dataUsingEncoding_preservingBOM() {
         func roundTrip(_ data: Data) -> Bool {
             let str = String(data: data, encoding: .utf8)!
-            let strAsUTF16BE = str.data(using: .utf16BigEndian)!
+            let strAsUTF16BE = str.data(using: String._Encoding.utf16BigEndian)!
```

Comment:
Hmm I wonder why are they suddenly an issue now (as in you have to spill out `String._Encoding`)...

Reply:
It's because `String.data(using:)` was added in macOS 15 (only `StringProtocol.data(using:)` existed before that version) so previously the `String.data(using:)` from the package was favored over the `StringProtocol.data(using:)` from the package and from the SDK, but now that `String.data(using:)` is both in the package and in the SDK it becomes ambiguous